lib/game_stats.rb

```diff
@@ -48,9 +49,19 @@ def percentage_visitor_wins
       if game[:away_goals] > game[:home_goals]
       winners += 1
         end
-        binding.pry
+        #inding.pry
 
       end
     (100 * winners.to_f/@all_games.length).round(2)
   end
+
+  def average_goals_per_game
+    total_scores = 0
+    @all_games.each do |game|
+      total_scores += (game[:away_goals] + game[:home_goals])
+      #binding.pry
+      end
+      (total_scores/50.00)
```

Comment:
I don't think we should use a hard-coded number here. There are 50 total games in our test data set but we'll need this to be scaleable for the main data set.